### ISSUE TITLE
[Bug] No such image: moby/buildkit:buildx-stable-1

### ISSUE THREAD

**Issue by litany-lty** [jammsen/docker-palworld-dedicated-server#30]
### Have you read the Important information text above

- [X] Yes i did

### Current behavior

![image](https://github.com/jammsen/docker-palworld-dedicated-server/assets/57203670/f3dd470e-8f03-47b9-9d13-8f0bf8e5d5b8)
[steam@VM-4-15-centos ~]$ vi docker-compose.yml 
[steam@VM-4-15-centos ~]$ docker-compose up -d
[+] Running 0/1
 ? palworld Pulling                                                                                                                                                                                                                                                                                                                    2.5s 
[+] Building 2.6s (1/1) FINISHED                                                                                                                                                                                                                                                                                   docker-container:default
 => ERROR [palworld internal] booting buildkit                                                                                                                                                                                                                                                                                         2.6s
 => => pulling image moby/buildkit:buildx-stable-1                                                                                                                                                                                                                                                                                     2.6s
 => => creating container buildx_buildkit_default                                                                                                                                                                                                                                                                                      0.0s
------
 > [palworld internal] booting buildkit:
------
Error response from daemon: No such image: moby/buildkit:buildx-stable-1
[steam@VM-4-15-centos ~]$ 


### Desired behavior

这个镜像依赖的相关东西我无法下载

### Links to screenshots

_No response_

### To Reproduce

[steam@VM-4-15-centos ~]$ vi docker-compose.yml 
[steam@VM-4-15-centos ~]$ docker-compose up -d
[+] Running 0/1
 ? palworld Pulling                                                                                                                                                                                                                                                                                                                    2.5s 
[+] Building 2.6s (1/1) FINISHED                                                                                                                                                                                                                                                                                   docker-container:default
 => ERROR [palworld internal] booting buildkit                                                                                                                                                                                                                                                                                         2.6s
 => => pulling image moby/buildkit:buildx-stable-1                                                                                                                                                                                                                                                                                     2.6s
 => => creating container buildx_buildkit_default                                                                                                                                                                                                                                                                                      0.0s
------
 > [palworld internal] booting buildkit:
------
Error response from daemon: No such image: moby/buildkit:buildx-stable-1
[steam@VM-4-15-centos ~]$ 


### Software setup

。。


### Hardware setup

..

### Additional context

_No response_

**Comment by jammsen** [comment]
Hey @litany-lty 
Im confused  are you trying to run the finished docker-image or to build yourself?
You should do a little more then just copy paste your error. Please describe what you trying to do and what happend and how your setup is. The fields Software-Setup and Hardware-Setup are there for a reason, not to annoy you, but to help me to understand you, do not just insert "..", thats of no help.

**Comment by litany-lty** [comment]
> 嘿@litany-lty 我很困惑你是想运行完成的 docker-image 还是自己构建？ 您应该多做一点，然后复制粘贴您的错误。请描述您想要做什么、发生了什么以及您的设置如何。软件设置和硬件设置字段的存在是有原因的，不是为了惹恼你，而是为了帮助我理解你，不要只插入“..”，那没有任何帮助。

"Sorry, this issue was caused by different system versions. Initially, I was unable to successfully pull the image because I'm in China and the proxy server was not effective. Later on, while executing 'docker-compose up -d', the image was being pulled. Then an error occurred saying 'Trying to pull repository docker.io/moby/buildkit... missing signature key'.
I was unable to deploy the server through containerization, however, it was strange that it worked when I manually installed steamCMD and PalServer to host the game server without the network proxy.
Below are the system details: NAME="CentOS Linux" VERSION="7 (Core)" ID="centos" ID_LIKE="rhel fedora" VERSION_ID="7" PRETTY_NAME="CentOS Linux 7 (Core)" ANSI_COLOR="0;31" CPE_NAME="cpe:/o:centos:centos:7" HOME_URL="https://www.centos.org/" BUG_REPORT_URL="https://bugs.centos.org/"
CENTOS_MANTISBT_PROJECT="CentOS-7" CENTOS_MANTISBT_PROJECT_VERSION="7" REDHAT_SUPPORT_PRODUCT="centos" REDHAT_SUPPORT_PRODUCT_VERSION="7"
 
But in actuality, it is CentOS 7.9 64bit from a Tencent server.
